src/components/ChatEntry.js

```diff
@@ -2,21 +2,48 @@ import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
+const yearsAgo = (string) => {
+  const yearsAgo = 2022 - parseInt(string.slice(0, 4), 10);
+  return yearsAgo;
+};
+
 const ChatEntry = (props) => {
+  const onLikeButton = () => {
+    const updatedEntry = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked
+    };
+    props.onLike(updatedEntry);
+  };
+  
+  const likeHeart = props.liked ? '❤️' : '🤍';
+  const placement = props.sender === 'Vladimir' ? 'local' : 'remote'
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={`chat-entry ${placement}`}>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+        <p className="entry-time">
+          {yearsAgo(props.timeStamp)} {'years ago'}
```

Comment:
You can use the provided [TimeStamp component](https://github.com/Ada-C18/react-chatlog/blob/main/src/components/TimeStamp.js) and pass it props.timeStamp to correctly rendered the elapsed time part of the ChatEntry